emission/net/ext_service/habitica/sync_habitica.py

```diff
@@ -0,0 +1,68 @@
+# Standard imports
+import json
+import requests
+import logging
+import arrow
+import attrdict as ad
+
+# Our imports
+import emission.core.get_database as edb
+import emission.net.ext_service.habitica.proxy as proxy
+import emission.analysis.result.metrics.simple_metrics as earmts
+import emission.analysis.result.metrics.time_grouping as earmt
+
+
+
+def create_habit(user_id, new_habit):
+  method_uri = "/api/v3/tasks/user"
+  #how to pass in params? https://habitica.com/apidoc/#api-Task-GetUserTasks
+  #FIX! right now it's returning all types of tasks, I want it to return only "habits"
+  params = {'type': "habits"}
+  result = proxy.habiticaProxy(user_id, 'GET', method_uri, params)
+  habits = result.json()
+  for habit in habits['data']:
+    if habit['text'] == new_habit['text']:
+      return habit['_id']
+  #if habit not found, create habit
+  response = proxy.habiticaProxy(user_id, 'POST', method_uri, new_habit)
+  habit_created = response.json()
+  return habit_created['data']['_id']
+
+
+
+def reward_active_transportation(self, user_id):
+  #make sure habits exist
+  #bike
+  bike_habit = {'type': "habit", 'text': "Bike", 'up': True, 'down': False, 'priority': 2}
+  bike_habit_id = create_habit(user_id, bike_habit)
+  #walk
+  walk_habit = {'type': "habit", 'text': "Walk", 'up': True, 'down': False, 'priority': 2}
+  walk_habit_id = create_habit(user_id, walk_habit)
+
+  #get timestamps
+  user_val = list(edb.get_habitica_db().find({"user_id": user_id}))[0]
+  timestamp_from_db = user_val['last_timestamp']
+  timestamp_now = arrow.utcnow().timestamp
+  
+  #FIX! how do I change the frequency to be just one segment? For now I just left it as yearly
+  summary_ts = earmt.group_by_timestamp(user, timestamp_from_db, timestamp_now, earmt.LocalFreq.YEARLY, earmts.get_distance)
```

Comment:
Are you sure this works?
IIRC, for `group_by_timestamp` you pass in the pandas dateoffset strings.

`The frequency as specified in a pandas date_range frequency string.
    We only support frequencies of a day or longer in order to return the data
    in a format that makes sense`

If you want only one segment, year seems like a good way to get it, given that this script runs every hour.